src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java

```diff
@@ -338,10 +339,14 @@ private PersonalAccessToken getWebHookCredentials(SCMSourceOwner owner) {
             String fullName = gitlabOwner.getFullName();
             String webUrl = gitlabOwner.getWebUrl();
             String avatarUrl = gitlabOwner.getAvatarUrl();
+            String description = null;
+            if (gitlabOwner instanceof GitLabGroup) {
+                description = ((GitLabGroup) gitlabOwner).getDescription();
+            }
             List<Action> result = new ArrayList<>();
             result.add(new ObjectMetadataAction(
                 Util.fixEmpty(fullName),
```

Comment:
The display name of GitLab Group is not updated but the description is updated. :/

Reply:
what? 😭
So you can get some absurd long names 😆

Reply:
@baymac if user has given the group a display name or description it does not override

Reply:
This works for groups: 
![image](https://user-images.githubusercontent.com/1661688/63983122-a0674000-cac5-11e9-9a5d-6c89293379d2.png)